src/content/docs/en/guides/cms/payload.mdx

```diff
@@ -217,7 +217,8 @@ export async function getStaticPaths() {
 
 To deploy your site visit our [deployment guide](/en/guides/deploy/) and follow the instructions for your preferred hosting provider.
 
-## Community Resources 
+## Community Resources
 
+- Check out the [official Astro integration](https://github.com/payloadcms/payload/tree/main/examples/astro) on the Payload CMS repository
```

Comment:
```suggestion
- Check out the [official Astro Payload CMS integration](https://github.com/payloadcms/payload/tree/main/examples/astro).
```
The link text should be as descriptive as possible about what's at the link and where it's going. What about this?

Reply:
perfect, thank you! 🫶